doc/theory/glossary.tex

```diff
@@ -9,9 +9,9 @@
 	\section{Symbols}
 	
 	\begin{tabular}{ l l l}
-		$\Bbbk$ & :  & An arbitrary field. \\
+		$\Bbbk$ & : & An arbitrary field. \\
 		$\mathbb R^d$ & : & The vector space of $d$-tuples of real numbers. \\
-		$\mathbb N$ & : & The counting numbers $0,1,2, \dots$ as a subset of $\mathbb R$. \\
+		$\mathbb N_{(>0)}$ & : & The counting numbers $(0, )1, 2, \ldots$ as a subset of $\mathbb R$. \\
```

Comment:
I think the \ldots look good the other change not so much. Do you have a specific need for this extra notation?

Reply:
@ammedmar my bad. There was a need at one point in this PR, but no more, so it should be removed.

Reply:
Fixed.